functions/index.js

```diff
@@ -21,10 +25,24 @@ const gerrit = new Gerrit(
   functions.config().gerrit.user,
   functions.config().gerrit.password)
 
+const crowd = new Crowd(
+  db,
+  functions.config().crowd.app_name,
+  functions.config().crowd.app_password)
+
+const crowdWebhook = new CrowdWebhook(
+  functions.config().crowdGroupMappings,
```

Comment:
Are we sure we can use upper case for config keys? Also, did we already deploy/set this config?

Reply:
I didn't try this.. I just used the default fallback. You are probably right